src/Appwrite/Platform/Workers/Deletes.php

```diff
@@ -412,14 +412,22 @@ function (Document $document) use ($cache, $projectId) {
      * @return void
      * @throws Exception
      */
-    private function deleteUsageStats(Document $project, callable $getProjectDB, string $hourlyUsageRetentionDatetime): void
+    private function deleteUsageStats(Document $project, callable $getProjectDB, callable $getLogsDB, string $hourlyUsageRetentionDatetime): void
     {
         $dbForProject = $getProjectDB($project);
-        // Delete Usage stats
+        $dbForLogs = $getLogsDB($project);
+
+        // Delete Usage stats from projectDB
         $this->deleteByGroup('stats', [
             Query::lessThan('time', $hourlyUsageRetentionDatetime),
             Query::equal('period', ['1h']),
         ], $dbForProject);
+
+        // Delete Usage stats from logsDB
+        $this->deleteByGroup('stats', [
```

Comment:
okay this is usage stats. We also need to delete the audit logs ( activities ) on the new logs DB